src/components/NavBar/NavBar.vue

```diff
@@ -29,13 +29,14 @@
           hide-no-data
           clearable
           click:clear
+          data-cy="autocomplete-search-bar"
         >
           <template v-slot:item="data">
-            <v-list-tile-content
-            v-on:click="redirect(data.item['symbol'])"
+            <v-list-item-content
+            data-cy="autocomplete-list-content"
             >
-              <v-list-tile-title ><strong>{{ data.item['symbol'] }}</strong>: {{ data.item['name'] }}</v-list-tile-title>
-            </v-list-tile-content>
+              <v-list-item-title v-on:click="redirect(data.item['symbol'])" data-cy="autocomplete-list-item" ><strong>{{ data.item['symbol'] }}</strong>: {{ data.item['name'] }}</v-list-item-title>
```

Comment:
you can bind with only : now, you don't need v-on:
```suggestion
              <v-list-item-title :click="redirect(data.item['symbol'])" data-cy="autocomplete-list-item" ><strong>{{ data.item['symbol'] }}</strong>: {{ data.item['name'] }}</v-list-item-title>
```

Reply:
I'll mark that somewhere so we can make that change in the next set of refactors.